### DISCUSSION TITLE
(Softcoding / Mods folder) Note size help. [0.4.0]

### DISCUSSION THREAD

**Discussion by jupitorfish** [FunkinCrew/Funkin#2948] [Q&A] [answered]
Hi, Can someone help me? Im using a duplicate of the funkin' notetype json within the mods folder, and for some reason when i change the scale of the notes, it only affects the static notes and not the moving / saturated arrows, anyone got a fix for this? thanks very much. in addition i am using the x64 0.4.0 build directly off the github, with no outside hard-coding, :) -

https://github.com/FunkinCrew/Funkin/assets/169615204/86a3dd37-ff38-4003-b4e4-b5096a566213

![image](https://github.com/FunkinCrew/Funkin/assets/169615204/ea03df64-398c-4a4f-a02a-05fb046c6269)

here is the code for the note json file (dx.json)
`{
  "version": "1.0.0",
  "name": "dx",
  "author": "me",
  "fallback": null,
  "assets": {
    "note": {
      "assetPath": "shared:notes",
      "scale": 1,
      "data": {
        "left": { "prefix": "noteLeft" },
        "down": { "prefix": "noteDown" },
        "up": { "prefix": "noteUp" },
        "right": { "prefix": "noteRight" }
      }
    },
    "noteStrumline": {
      "assetPath": "shared:noteStrumline",
      "scale": 1.0,
      "offsets": [0, 0],
      "data": {
        "leftStatic": { "prefix": "staticLeft0" },
        "leftPress": { "prefix": "pressLeft0" },
        "leftConfirm": { "prefix": "confirmLeft0" },
        "leftConfirmHold": { "prefix": "confirmLeft0" },
        "downStatic": { "prefix": "staticDown0" },
        "downPress": { "prefix": "pressDown0" },
        "downConfirm": { "prefix": "confirmDown0" },
        "downConfirmHold": { "prefix": "confirmDown0" },
        "upStatic": { "prefix": "staticUp0" },
        "upPress": { "prefix": "pressUp0" },
        "upConfirm": { "prefix": "confirmUp0" },
        "upConfirmHold": { "prefix": "confirmUp0" },
        "rightStatic": { "prefix": "staticRight0" },
        "rightPress": { "prefix": "pressRight0" },
        "rightConfirm": { "prefix": "confirmRight0" },
        "rightConfirmHold": { "prefix": "confirmRight0" }
      }
    },
    "holdNote": {
      "assetPath": "NOTE_hold_assets",
      "data": {}
    },
    "noteSplash": {
      "assetPath": "",
      "data": {
        "enabled": true
      }
    },
    "holdNoteCover": {
      "assetPath": "",
      "data": {
        "enabled": true
      }
    }
  }
}
`

**Comment by Lasercar** [comment, marked as answer]
That's because you need to set the scale for the strumline _and_ the notes. Take note from how the pixel note style sets the scale.
